dev/.gitignore

```diff
@@ -17,4 +17,5 @@
 
 # Python virtual environments for dev tools
 .venv*/
+venv/
```

Comment:
Why not reuse the same `.venv` convention as above?

Reply:
both are common uses, right?

Reply:
That's true. If this is for personal use you may want to broaden it to `venv*/`, then.